src/aiq_agent/agents/deep_researcher/skills/visualization/chart-generation/SKILL.md

```
@@ -167,3 +192,51 @@ filename in the report and do not invent or rename it later.
+`line`/`area` (a trend across an ordered axis), `grouped-bar` (2-4 series per category),
+`delta` (gains vs losses around zero).
+
+Spec fields: `type`; `title` (short) and optional `subtitle`; `x` = `{ "key": "<field in each
+row>", "label": "optional" }`; optional `y` = `{ "label": "optional unit", "format": "number |
+compact | percent | currency" }`; `series` = `[ { "key": "<numeric field>", "label": "optional",
+"color": "green | blue | amber | red" } ]`; `data` = rows as objects with raw numbers (fractions
+0-1 for `percent`); optional `kpis` = `[ { "label": "...", "value": "preformatted", "tone": "accent
+| warn | alarm" } ]`. A `delta` chart encodes exactly one series.
+
+Example (ranking):
+```chart
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)